src/messages/MMgrBeacon.h

```diff
@@ -169,7 +179,23 @@ class MMgrBeacon final : public PaxosServiceMessage {
       decode(mgr_features, p);
     }
     if (header.version >= 10) {
-      decode(clients, p);
+      std::vector<entity_addrvec_t> clients_addrs;
+      decode(clients_addrs, p);
+      clients.clear();
+      if (header.version >= 11) {
+	std::vector<std::string> clients_names;
+	decode(clients_names, p);
+	assert(clients_names.size() == clients_addrs.size());
+	auto cn = clients_names.begin();
+	auto ca = clients_addrs.begin();
+	for(; cn!=clients_names.end() && ca!=clients_addrs.end(); ++cn, ++ca) {
```

Comment:
Since the sizes were validated, checking just one of the ends is sufficient. Also formatting nits:
```suggestion
	for( ; cn != clients_names.end(); ++cn, ++ca) {
```